rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -23,6 +23,30 @@ import cdm.product.collateral.*
 
 import cdm.mapping.config.*
 
+choice Asset:  <"An Asset is defined something that can be owned and transferred in the financial markets. As a choice data type, one and only one of the  attributes must be used. Cash: An Asset that consists solely of a monetary holding in a currency. Commodity: An Asset comprised of raw or refined materials or agricultural products, eg gold, oil or wheat. DigitalAsset An Asset that exists only in digital form, eg Bitcoin or Ethereum; excludes the digital representation of other Assets. Instrument: An asset that is issued by one party to one or more others; Instrument is also a choice data type.">
+    Cash
+    Commodity
```

Comment:
When `Commodity` eventually extends `AssetBase` instead of `ProductBase`, it will need a `Taxonomy` attribute (but no need for the fuller `ProductTaxonomy`). Hence consider adding an optional `taxonomy` attribute to all `Asset` and not just `Instrument`.